h/admin.py

```diff
@@ -260,7 +308,6 @@ def _all_user_annotations_query(userid):
         }
     }
 
-
```

Comment:
As mentioned earlier, this two-blank-lines things is weird, but it is the convention...